CHANGELOG.md

```diff
@@ -5,13 +5,14 @@
 ### ✨ New features
 
 - *...Add new stuff here...*
-- Bump maplibre-native-base from 2.0.0 to 2.1.0 [#397](https://github.com/maplibre/maplibre-gl-native/pull/397)
+- Bump [maplibre-native-base](https://github.com/maplibre/maplibre-native-base) from 2.0.0 to 2.1.0 [#397](https://github.com/maplibre/maplibre-gl-native/pull/397)
+- Bump [wagyu](https://github.com/mapbox/wagyu) from 0.4.3 to 0.5.0
```

Comment:
Should we make a new category for such bumps? I'm not sure if these are new features really as these dependencies are mostly not exposed to users.

Reply:
that's true. in the other the category is just called features and improvements

Reply:
Maybe call it "Technical improvements" as it may still be nice to have features separate.

Also we should figure out what else could be added to the changelogs but we have just forgotten :)

Reply:
If it's okay I can make this alteration to the changelog in the next pr, so that the github runneres doesn't have to start all over

Reply:
There's a conflict anyways (sorry :( ). I will not merge anything else in the near future, so feel free to rebase.

Reply:
No worries, these updates could take days, so it shouldn't block everything else